Scheduler: Update Editing and Templates demo descriptions (#31939)
DevExpress/DevExtreme: apps/demos/Demos/Scheduler/Templates/description.md

You can double-click a DevExtreme Scheduler appointment or an empty cell to open an edit form from the UI. Scheduler allows you to customize the form: rearrange items, create custom fields, and specify popup settings.

> The form includes two groups: `mainGroup` (general information) and `recurrenceGroup` (recurrence settings). Scheduler displays `mainGroup` first and switches to `recurrenceGroup` when users change the "Repeat" drop-down. Configure **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) to customize the form layout and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize the dialog window. This demo configures these objects to add custom fields to the edit form.

Reviewer: I would probably be more specific:

```suggestion
The form includes two groups: `mainGroup` (general information) and `recurrenceGroup` (recurrence settings). Scheduler displays `mainGroup` first and switches to `recurrenceGroup` when users change the "Repeat" drop-down. Configure **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) to customize the form layout and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize the dialog window. This demo configures these objects to add custom fields (**Field1** and **Field2**) to the edit form.
```

Reviewer 2: ```suggestion
The form includes two groups: `mainGroup` (general information) and `recurrenceGroup` (recurrence settings). Scheduler displays `mainGroup` first and switches to `recurrenceGroup` when users change the "Repeat" drop-down. Configure **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) to customize the form layout and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize the dialog window. This demo configures these objects to add custom fields (**movieId** and **price**) to the edit form.
```

> The form includes two groups: `mainGroup` (general information) and `recurrenceGroup` (recurrence settings). Scheduler displays `mainGroup` first and switches to `recurrenceGroup` when users change the "Repeat" drop-down. Configure **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize the form layout.

Reviewer 3: Maybe 
Configure **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/popup/) to customize edit form's dialog options and **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) to customize the form's layout.

> For more details, refer to the [Appointment Edit Form](/Documentation/Guide/UI_Components/Scheduler/Appointment/Appointment_Edit_Form/) help topic.

Reviewer: 100% Ray would edit this as follows:

```suggestion
For additional information, refer to the following help topic: [Appointment Edit Form](/Documentation/Guide/UI_Components/Scheduler/Appointment/Appointment_Edit_Form/).
```

> [note] Image Source: **Wikimedia Commons**

Reviewer 3: Let's place this after everything else
